docker-compose.yml

```
@@ -15,4 +15,11 @@ services:
     image: peoplepartyportal
     build:
       context: ./frontend
-      dockerfile: dockerfile
\ No newline at end of file
+      dockerfile: dockerfile
+  api:
+    image: peoplepartyserver
+    build:
+      context: ./backend
+      dockerfile: dockerfile
+    ports:
+      - 7789:80
```

We might want some restart policy here, because it's possible for the backend to fall through and exit.

```suggestion
  api:
    image: peoplepartyserver
    build:
      context: ./backend
      dockerfile: dockerfile
    ports:
      - 7789:80
    restart: unless-stopped
```